org.qutebrowser.qutebrowser.yml

```diff
@@ -0,0 +1,43 @@
+---
+app-id: org.qutebrowser.qutebrowser
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: qutebrowser
+rename-desktop-file: qutebrowser.desktop
+rename-icon: qutebrowser
+finish-args:
+- "--share=ipc"
+- "--socket=x11"
+- "--socket=wayland"
+- "--socket=pulseaudio"
+- "--share=network"
+- "--filesystem=xdg-download"
+- "--device=dri"
+cleanup:
+- "/lib/debug"
+- "/lib/libgudev*"
+modules:
+- shared-modules/udev/udev-175.json
+- pygments.yml
+- pypeg2.yml
+- markupsafe.yml
+- jinja2.yml
+- attrs.yml
+- pyyaml.yml
+- sip.yml
+- pyqt5.yml
+- name: qutebrowser
+  buildsystem: simple
+  build-commands:
+  - pip3 install --no-index --no-compile --prefix=/app qutebrowser-1.5.2.tar.gz
+  - tar xf qutebrowser-1.5.2.tar.gz
+  - install -Dm644 qutebrowser-1.5.2/misc/qutebrowser.appdata.xml /app/share/metainfo/qutebrowser.appdata.xml
```

Comment:
The appdata file should be installed by the build system like the desktop file & icons.